ibis/backends/dask/execution/util.py

```diff
@@ -272,9 +272,7 @@ def compute_sorted_frame(
             new_columns[computed_sort_key] = temporary_column
 
     result = df.assign(**new_columns)
-    result = result.sort_values(
-        computed_sort_keys, ascending=ascending, kind="mergesort"
```

Comment:
I removed the use of `kind="mergesort"` here because this is apparently not supported in Dask anymore (as of a recent release I guess?).